crates/ln-dlc-node/src/node/oracle_client.rs

```diff
@@ -4,12 +4,11 @@ use dlc_manager::Oracle;
 use p2pd_oracle_client::P2PDOracleClient;
 use std::str::FromStr;
 
-// TODO: This should come from the configuration.
-const ORACLE_ENDPOINT: &str = "https://oracle.holzeis.me/";
+pub fn build(oracle_endpoint: String) -> P2PDOracleClient {
+    // TODO: fetch public key from oracle at ORACLE_ENDPOINT/oracle/publickey
```

Comment:
``` suggestion
```

I don't think we should ever fetch the pubkey from the oracle itself but the pubkey should be hardcoded because it should not change; this comment should be removed.

Reply:
It is hardcoded.